index.yaml

```diff
@@ -14,4 +27,4 @@ entries:
     urls:
     - https://sematic-ai.github.io/helm-charts/sematic-server/sematic-server-1.0.0.tgz
     version: 1.0.0
-generated: "2023-01-07T21:41:53.247214-08:00"
```

Comment:
undo this hunk, we don't want to update timestamps of older releases.  i'll discuss with you tomorrow about why this happened.

Reply:
nvm i'm wrong, please ignore